src/js/components/Drop/DropContainer.js

```diff
@@ -343,11 +343,20 @@ const DropContainer = forwardRef(
       dropOptions,
     ]);
 
+    // Once drop is open the focus will be put on the drop container
+    // if restrictFocus is true. If the caller put focus
+    // on an element already, we honor that. Otherwise, we put
+    // the focus on the drop container.
     useEffect(() => {
       if (restrictFocus) {
-        dropRef.current.focus();
+        const dropElement = dropRef.current;
```

Comment:
Small naming suggestion to align with component:

- dropElement --> dropContainer

Reply:
can switch!